rtos/TARGET_CORTEX/mbed_rtx_conf.h

```diff
@@ -56,4 +56,20 @@
 # define OS_PRIVILEGE_MODE           0
 #endif
 
+#if defined (__ARM_FEATURE_CMSE) &&  (__ARM_FEATURE_CMSE == 3U)
+
+/* Number of process slots (threads may call secure library code) */
+#ifndef MBED_CONF_APP_TZ_PROCESS_STACK_SLOTS
+#define MBED_CONF_APP_TZ_PROCESS_STACK_SLOTS     8U
```

Comment:
why 8? Can we document that

Reply:
Ideally it should be number of threads we allow in mbed-os. (Considering all of them have access to secure library). Or do we want to restrict threads from using secure library?